--- tk8.4.7/unix/tcl.m4.orig	2004-08-06 00:22:26.877855448 +0200
+++ tk8.4.7/unix/tcl.m4	2004-08-06 00:26:55.430029312 +0200
@@ -904,7 +904,7 @@
 
 	    # AIX v<=4.1 has some different flags than 4.2+
 	    if test "$system" = "AIX-4.1" -o "`uname -v`" -lt "4" ; then
-		LIBOBJS="$LIBOBJS tclLoadAix.o"
+		AC_LIBOBJ([tclLoadAix])
 		DL_LIBS="-lld"
 	    fi
 
@@ -1162,7 +1162,7 @@
 	    # is kind of overkill but it works.
 	    # Disable inlining only when one of the
 	    # files in compat/*.c is being linked in.
-	    if test x"${LIBOBJS}" != x ; then
+	    if test x"${LIB@&t@OBJS}" != x ; then
 	        CFLAGS="$CFLAGS -fno-inline"
 	    fi
 
@@ -2283,7 +2283,7 @@
 	    AC_MSG_RESULT(ok)
 	else
 	    AC_MSG_RESULT(buggy)
-	    LIBOBJS="$LIBOBJS fixstrtod.o"
+	    AC_LIBOBJ([fixstrtod])
 	    AC_DEFINE(strtod, fixstrtod)
 	fi
     fi
