diff -Nru tk8.0.5/generic/tkPort.h tk8.0.5.new/generic/tkPort.h
--- tk8.0.5/generic/tkPort.h	Mon Sep 14 20:23:16 1998
+++ tk8.0.5.new/generic/tkPort.h	Mon Sep  6 14:14:22 1999
@@ -29,7 +29,7 @@
 #   if defined(MAC_TCL)
 #	include "tkMacPort.h"
 #   else
-#	include "../unix/tkUnixPort.h"
+#	include <tkUnixPort.h>
 #   endif
 #endif
 
diff -Nru tk8.0.5/unix/Makefile.in tk8.0.5.new/unix/Makefile.in
--- tk8.0.5/unix/Makefile.in	Tue Mar  9 07:53:50 1999
+++ tk8.0.5.new/unix/Makefile.in	Mon Sep  6 14:13:31 1999
@@ -396,7 +396,17 @@
 INSTALL_HDRS = $(GENERIC_DIR)/tk.h  \
 	$(GENERIC_DIR)/tkInt.h \
 	$(GENERIC_DIR)/tkDecls.h \
-	$(GENERIC_DIR)/tkIntXlibDecls.h
+	$(GENERIC_DIR)/tkIntDecls.h \
+	$(GENERIC_DIR)/tkCanvas.h \
+	$(GENERIC_DIR)/tkIntXlibDecls.h \
+	$(GENERIC_DIR)/tkPort.h \
+	$(GENERIC_DIR)/default.h \
+	$(UNIX_DIR)/tkUnixDefault.h \
+	$(UNIX_DIR)/tkUnixInt.h \
+	$(UNIX_DIR)/tkUnixPort.h \
+	$(COMPAT_DIR)/limits.h \
+	$(COMPAT_DIR)/unistd.h \
+	$(COMPAT_DIR)/stdlib.h \
 
 DEMOPROGS = browse hello ixset rmt rolodex square tcolor timer widget
 
