diff -urN tk8.5a3.org/unix/Makefile.in tk8.5a3/unix/Makefile.in
--- tk8.5a3.org/unix/Makefile.in	2005-06-04 23:00:25.000000000 +0200
+++ tk8.5a3/unix/Makefile.in	2005-06-17 23:54:19.497558856 +0200
@@ -677,7 +677,13 @@
 	    fi
 	@echo "Installing header files";
 	@for i in $(GENERIC_DIR)/tk.h $(GENERIC_DIR)/tkDecls.h \
-		$(GENERIC_DIR)/tkPlatDecls.h $(@TK_WINDOWINGSYSTEM@_HDRS) ; \
+		$(GENERIC_DIR)/tkPlatDecls.h $(@TK_WINDOWINGSYSTEM@_HDRS) \
+		$(GENERIC_DIR)/tkIntXlibDecls.h \
+		$(GENERIC_DIR)/tkIntDecls.h $(GENERIC_DIR)/tkCanvas.h \
+		$(UNIX_DIR)/tkUnixPort.h $(GENERIC_DIR)/tkPort.h \
+		$(GENERIC_DIR)/default.h $(UNIX_DIR)/tkUnixDefault.h \
+		$(UNIX_DIR)/tkUnixInt.h \
+		; \
 	    do \
 	    $(INSTALL_DATA) $$i $(INCLUDE_INSTALL_DIR); \
 	    done;
