--- tk8.4.3/unix/Makefile.in.wiget	2003-05-20 20:37:04.000000000 +0200
+++ tk8.4.3/unix/Makefile.in	2003-06-11 09:15:18.000000000 +0200
@@ -563,7 +563,13 @@
 	    fi
 	@echo "Installing header files";
 	@for i in $(GENERIC_DIR)/tk.h $(GENERIC_DIR)/tkDecls.h \
-		$(GENERIC_DIR)/tkPlatDecls.h ; \
+		$(GENERIC_DIR)/tkPlatDecls.h  \
+ 		$(GENERIC_DIR)/tkIntXlibDecls.h \
+ 		$(GENERIC_DIR)/tkIntDecls.h $(GENERIC_DIR)/tkCanvas.h \
+ 		$(UNIX_DIR)/tkUnixPort.h $(GENERIC_DIR)/tkPort.h \
+ 		$(GENERIC_DIR)/default.h $(UNIX_DIR)/tkUnixDefault.h \
+ 		$(UNIX_DIR)/tkUnixInt.h $(COMPAT_DIR)/limits.h \
+ 		$(COMPAT_DIR)/unistd.h $(COMPAT_DIR)/stdlib.h ; \
 	    do \
 	    $(INSTALL_DATA) $$i $(INCLUDE_INSTALL_DIR); \
 	    done;

