--- tk8.4.7/unix/configure.in.orig	2004-07-29 23:57:48.000000000 +0200
+++ tk8.4.7/unix/configure.in	2004-08-06 00:22:07.871744816 +0200
@@ -129,13 +129,10 @@
 #--------------------------------------------------------------------
 # On a few very rare systems, all of the libm.a stuff is
 # already in libc.a.  Set compiler flags accordingly.
-# Also, Linux requires the "ieee" library for math to work
-# right (and it must appear before "-lm").
 #--------------------------------------------------------------------
 
 MATH_LIBS=""
 AC_CHECK_FUNC(sin, , MATH_LIBS="-lm")
-AC_CHECK_LIB(ieee, main, [MATH_LIBS="-lieee $MATH_LIBS"])
 
 #--------------------------------------------------------------------
 # On AIX systems, libbsd.a has to be linked in to support
