--- tk8.0/unix/configure.in.ieee	Wed Oct 22 13:44:22 1997
+++ tk8.0/unix/configure.in	Wed Oct 22 13:44:29 1997
@@ -285,13 +285,10 @@
 #--------------------------------------------------------------------
 #	On a few very rare systems, all of the libm.a stuff is
 #	already in libc.a.  Set compiler flags accordingly.
-#	Also, Linux requires the "ieee" library for math to
-#	work right (and it must appear before "-lm").
 #--------------------------------------------------------------------
 
 MATH_LIBS=""
 AC_CHECK_FUNC(sin, , MATH_LIBS="-lm")
-AC_CHECK_LIB(ieee, main, [MATH_LIBS="-lieee $MATH_LIBS"])
 
 #--------------------------------------------------------------------
 #	If this system doesn't have a memmove procedure, use memcpy
