diff -urBb tk8.4.11.org/unix/Makefile.in tk8.4.11/unix/Makefile.in
--- tk8.4.11.org/unix/Makefile.in	2005-06-29 18:43:10.000000000 +0200
+++ tk8.4.11/unix/Makefile.in	2005-07-24 19:05:06.076871936 +0200
@@ -47,7 +47,7 @@
 # Directory from which applications will reference the library of Tcl
 # scripts (note: you can set the TK_LIBRARY environment variable at
 # run-time to override the compiled-in location):
-TK_LIBRARY		= @TK_LIBRARY@
+TK_LIBRARY		= $(libdir)/tk$(VERSION)
 
 # Path to use at runtime to refer to LIB_INSTALL_DIR:
 LIB_RUNTIME_DIR		= $(libdir)
