diff -urN tk8.5a3.org/unix/Makefile.in tk8.5a3/unix/Makefile.in
--- tk8.5a3.org/unix/Makefile.in	2005-06-04 23:00:25.000000000 +0200
+++ tk8.5a3/unix/Makefile.in	2005-06-17 23:59:45.195045312 +0200
@@ -47,7 +47,7 @@
 # Directory from which applications will reference the library of Tcl
 # scripts (note: you can set the TK_LIBRARY environment variable at
 # run-time to override the compiled-in location):
-TK_LIBRARY		= @TK_LIBRARY@
+TK_LIBRARY		= $(libdir)/tk$(VERSION)
 
 # Path to use at runtime to refer to LIB_INSTALL_DIR:
 LIB_RUNTIME_DIR		= $(libdir)
@@ -701,17 +701,7 @@
 	    rm -f "$(PKG_INDEX)"; \
 	    (\
 	    echo "if {[catch {package present Tcl 8.5.0}]} return";\
-	    relative=`echo | awk '{ORS=" "; split("$(TK_PKG_DIR)",a,"/"); for (f in a) {print ".."}}'`;\
-	    if test "x$(DLL_INSTALL_DIR)" != "x$(BIN_INSTALL_DIR)"; then \
-	    echo "package ifneeded Tk $(MAJOR_VERSION).$(MINOR_VERSION)$(PATCH_LEVEL) [list load [file normalize [file join \$$dir $${relative}$(TK_LIB_FILE)]] Tk]";\
-	    else \
-	    echo "if {(\$$::tcl_platform(platform) eq \"unix\") && ([info exists ::env(DISPLAY)]";\
-	    echo "	|| ([info exists ::argv] && (\"-display\" in \$$::argv)))} {";\
-	    echo "    package ifneeded Tk $(MAJOR_VERSION).$(MINOR_VERSION)$(PATCH_LEVEL) [list load [file normalize [file join \$$dir $${relative}.. bin $(TK_LIB_FILE)]] Tk]";\
-	    echo "} else {";\
-	    echo "    package ifneeded Tk $(MAJOR_VERSION).$(MINOR_VERSION)$(PATCH_LEVEL) [list load [file normalize [file join \$$dir $${relative}.. bin tk${MAJOR_VERSION}${MINOR_VERSION}.dll]] Tk]";\
-	    echo "}";\
-	    fi \
+	    echo "package ifneeded Tk $(MAJOR_VERSION).$(MINOR_VERSION)$(PATCH_LEVEL) [list load $(DLL_INSTALL_DIR)/$(TK_LIB_FILE) Tk]";\
 	    ) > "$(PKG_INDEX)"; \
 	    fi
 	@echo "Installing $(LIB_FILE) to $(DLL_INSTALL_DIR)/"
