--- tk8.6.12/unix/Makefile.in~	2021-11-04 16:06:24.000000000 +0100
+++ tk8.6.12/unix/Makefile.in	2021-11-11 15:09:46.142920245 +0100
@@ -281,8 +281,8 @@ TK_SHLIB_LD_EXTRAS	= @TK_SHLIB_LD_EXTRAS
 # Additional search flags needed to find the various shared libraries
 # at run-time.  The first symbol is for use when creating a binary
 # with cc, and the second is for use when running ld directly.
-CC_SEARCH_FLAGS	= @CC_SEARCH_FLAGS@
-LD_SEARCH_FLAGS	= @LD_SEARCH_FLAGS@
+CC_SEARCH_FLAGS	=
+LD_SEARCH_FLAGS	=
 
 # support for embedded libraries on Darwin / Mac OS X
 DYLIB_INSTALL_DIR	= $(libdir)
--- tk8.6.12/unix/tcl.m4~	2021-11-11 15:08:12.000000000 +0100
+++ tk8.6.12/unix/tcl.m4	2021-11-11 15:09:58.622920240 +0100
@@ -1991,7 +1991,7 @@ dnl # preprocessing tests use only CPPFL
 
     AS_IF([test "${SHARED_BUILD}" = 1 -a "${SHLIB_SUFFIX}" != ""], [
         LIB_SUFFIX=${SHARED_LIB_SUFFIX}
-        MAKE_LIB='${SHLIB_LD} -o [$]@ ${OBJS} ${LDFLAGS} ${SHLIB_LD_LIBS} ${TCL_SHLIB_LD_EXTRAS} ${TK_SHLIB_LD_EXTRAS} ${LD_SEARCH_FLAGS}'
+        MAKE_LIB='${SHLIB_LD} -o [$]@ ${OBJS} ${LDFLAGS} ${SHLIB_LD_LIBS} ${TCL_SHLIB_LD_EXTRAS} ${TK_SHLIB_LD_EXTRAS}'
         AS_IF([test "${SHLIB_SUFFIX}" = ".dll"], [
             INSTALL_LIB='$(INSTALL_LIBRARY) $(LIB_FILE) "$(BIN_INSTALL_DIR)/$(LIB_FILE)";if test -f $(LIB_FILE).a; then $(INSTALL_DATA) $(LIB_FILE).a "$(LIB_INSTALL_DIR)"; fi;'
             DLL_INSTALL_DIR="\$(BIN_INSTALL_DIR)"
