--- tk8.5a3.org/unix/Makefile.in	2005-06-04 23:00:25.000000000 +0200
+++ tk8.5a3/unix/Makefile.in	2005-06-17 23:57:52.837126312 +0200
@@ -262,8 +262,8 @@
 # Additional search flags needed to find the various shared libraries
 # at run-time.  The first symbol is for use when creating a binary
 # with cc, and the second is for use when running ld directly.
-CC_SEARCH_FLAGS	= @CC_SEARCH_FLAGS@
-LD_SEARCH_FLAGS	= @LD_SEARCH_FLAGS@
+CC_SEARCH_FLAGS	=
+LD_SEARCH_FLAGS	=
 
 # support for embedded libraries on Darwin / Mac OS X
 DYLIB_INSTALL_DIR	= ${LIB_RUNTIME_DIR}

--- tcl8.5.13/unix/tcl.m4.orig	2012-11-28 21:29:42.223960573 +0100
+++ tcl8.5.13/unix/tcl.m4	2012-11-28 21:31:03.440627511 +0100
@@ -2048,7 +2048,7 @@ dnl # preprocessing tests use only CPPFL
 
     AS_IF([test "${SHARED_BUILD}" = 1 -a "${SHLIB_SUFFIX}" != ""], [
         LIB_SUFFIX=${SHARED_LIB_SUFFIX}
-        MAKE_LIB='${SHLIB_LD} -o [$]@ ${OBJS} ${SHLIB_LD_LIBS} ${TCL_SHLIB_LD_EXTRAS} ${TK_SHLIB_LD_EXTRAS} ${LD_SEARCH_FLAGS}'
+        MAKE_LIB='${SHLIB_LD} -o [$]@ ${OBJS} ${SHLIB_LD_LIBS} ${TCL_SHLIB_LD_EXTRAS} ${TK_SHLIB_LD_EXTRAS}'
         AS_IF([test "${SHLIB_SUFFIX}" = ".dll"], [
             INSTALL_LIB='$(INSTALL_LIBRARY) $(LIB_FILE) "$(BIN_INSTALL_DIR)/$(LIB_FILE)"'
             DLL_INSTALL_DIR="\$(BIN_INSTALL_DIR)"

