--- tk8.4.11.org/unix/Makefile.in	2005-06-29 18:43:10.000000000 +0200
+++ tk8.4.11/unix/Makefile.in	2005-07-24 19:02:13.439116864 +0200
@@ -261,8 +261,8 @@
 # Additional search flags needed to find the various shared libraries
 # at run-time.  The first symbol is for use when creating a binary
 # with cc, and the second is for use when running ld directly.
-CC_SEARCH_FLAGS	= @CC_SEARCH_FLAGS@
-LD_SEARCH_FLAGS	= @LD_SEARCH_FLAGS@
+CC_SEARCH_FLAGS	=
+LD_SEARCH_FLAGS	=
 
 # support for embedded libraries on Darwin / Mac OS X
 DYLIB_INSTALL_DIR	= ${LIB_RUNTIME_DIR}
diff -urBb tk8.4.11.org/unix/tcl.m4 tk8.4.11/unix/tcl.m4
--- tk8.4.11.org/unix/tcl.m4	2005-06-29 18:43:10.000000000 +0200
+++ tk8.4.11/unix/tcl.m4	2005-07-24 19:03:19.934008104 +0200
@@ -1847,7 +1847,7 @@
 
     if test "${SHARED_BUILD}" = "1" && test "${SHLIB_SUFFIX}" != "" ; then
         LIB_SUFFIX=${SHARED_LIB_SUFFIX}
-        MAKE_LIB='${SHLIB_LD} -o [$]@ ${OBJS} ${SHLIB_LD_LIBS} ${TCL_SHLIB_LD_EXTRAS} ${TK_SHLIB_LD_EXTRAS} ${LD_SEARCH_FLAGS}'
+        MAKE_LIB='${SHLIB_LD} -o [$]@ ${OBJS} ${SHLIB_LD_LIBS} ${TCL_SHLIB_LD_EXTRAS} ${TK_SHLIB_LD_EXTRAS}'
         INSTALL_LIB='$(INSTALL_LIBRARY) $(LIB_FILE) $(LIB_INSTALL_DIR)/$(LIB_FILE)'
     else
         LIB_SUFFIX=${UNSHARED_LIB_SUFFIX}
