--- tk8.6.16/unix/tcl.m4.orig	2024-12-10 18:31:03.000000000 +0100
+++ tk8.6.16/unix/tcl.m4	2025-03-16 12:51:00.217260424 +0100
@@ -1073,7 +1073,7 @@ AC_DEFUN([SC_CONFIG_CFLAGS], [
     TCL_LIB_VERSIONS_OK=ok
     CFLAGS_DEBUG=-g
     AS_IF([test "$GCC" = yes], [
-	CFLAGS_OPTIMIZE=-O2
+	CFLAGS_OPTIMIZE="${OPTFLAGS}"
 	CFLAGS_WARNING="-Wall -Wpointer-arith"
     ], [
 	CFLAGS_OPTIMIZE=-O
@@ -1372,7 +1372,7 @@ AC_DEFUN([SC_CONFIG_CFLAGS], [
 	    SHLIB_CFLAGS="-fPIC -fno-common"
 	    SHLIB_SUFFIX=".so"
 
-	    CFLAGS_OPTIMIZE="-O2"
+	    CFLAGS_OPTIMIZE="${OPTFLAGS}"
 	    # egcs-2.91.66 on Redhat Linux 6.0 generates lots of warnings
 	    # when you inline the string and math operations.  Turn this off to
 	    # get rid of the warnings.
