--- tk8.4.6/unix/configure.in.orig	2004-03-06 22:34:25.000000000 +0100
+++ tk8.4.6/unix/configure.in	2004-03-06 22:39:05.193655656 +0100
@@ -363,7 +363,11 @@
     TCL_STUB_FLAGS=""
 fi
 TK_LIB_FILE=libtk${LIB_SUFFIX}
-eval "TK_LIB_FILE=${TK_LIB_FILE}"
+TK_SOMAJOR=".0"
+TK_SOVERSION="${TK_SOMAJOR}.0"
+eval "TK_LIB_SONAME=${TK_LIB_FILE}${TK_SOMAJOR}"
+eval "TK_LIB_FILE=${TK_LIB_FILE}${TK_SOVERSION}"
+AC_SUBST(TK_LIB_SONAME)
 
 # Note:  in the following variable, it's important to use the absolute
 # path name of the Tcl directory rather than "..":  this is because
--- tk8.4.6/unix/Makefile.in.orig	2004-03-06 22:34:25.000000000 +0100
+++ tk8.4.6/unix/Makefile.in	2004-03-06 22:37:05.308880912 +0100
@@ -404,7 +404,8 @@
 # a shared library or non-shared library for Tk.
 ${LIB_FILE}: ${OBJS}
 	rm -f $@
-	@MAKE_LIB@
+	@MAKE_LIB@ -Wl,-soname,@TK_LIB_SONAME@
+	-ln -sf ${TK_LIB_FILE} libtk${VERSION}@SHLIB_SUFFIX@
 
 ${STUB_LIB_FILE}: ${STUB_LIB_OBJS}
 	rm -f $@
